<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 208, in main
    return_code, report = execute_linter(
                          ~~~~~~~~~~~~~~^
        recipe=args.recipe,
        ^^^^^^^^^^^^^^^^^^^
    ...<6 lines>...
        verbose_flag=args.verbose,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 180, in execute_linter
    result = linter.lint(recipes, subdir, variant_config_files, exclusive_config_files, fix_flag)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 725, in lint
    msgs = self.lint_file(
        recipe_name,
    ...<3 lines>...
        fix=fix,
    )
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 831, in lint_file
    self.lint_recipe(
    ~~~~~~~~~~~~~~~~^
        recipe,
        ^^^^^^^
        fix=fix,
        ^^^^^^^^
    )
    ^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 891, in lint_recipe
    res = self.check_instances[check].run(recipe, fix)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 294, in run
    self.check_recipe(recipe)
    ~~~~~~~~~~~~~~~~~^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/check_syntax.py", line 36, in check_recipe
    has_constraints = constraints.search(spec)
TypeError: expected string or bytes-like object, got 'NoneType'

ERROR conda.cli.main_run:execute(127): `conda run conda-lint /tmp/abs_83ib_5qvew/clone` failed. (See above for error)
</details>

--- a/recipe/conda_build_config.yaml
+++ b/recipe/conda_build_config.yaml
@@ -1,10 +1,9 @@
-MACOSX_SDK_VERSION:                                          # [osx]
 # The required version for Qt is actually 12, but 0002-avoid-using-macos-12-resource.patch removes this requirement
 # and brings it back to v11.
-  - '11.3'                                                   # [osx]
-CONDA_BUILD_SYSROOT:                                         # [osx]
-  - /Library/Developer/CommandLineTools/SDKs/MacOSX11.3.sdk  # [osx]
-c_compiler:
-  - vs2019  # [win]
-cxx_compiler:
-  - vs2019  # [win]
+c_stdlib_version:  # [osx]
+  - 11.3           # [osx]
+CONDA_BUILD_SYSROOT:
+  - /opt/MacOSX11.3.sdk                                      # [osx and x86_64]
+  - /Library/Developer/CommandLineTools/SDKs/MacOSX11.3.sdk  # [osx and arm64]
+mysql:
+  - 8.4.*

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -18,8 +18,7 @@ source:
     folder: opengl32sw                                                                                         # [win]
 
 build:
-  number: 1
-  skip: True  # [ppc64le or s390x]
+  number: 2
   skip: True  # [osx and x86_64]
   run_exports:
     - {{ pin_subpackage(name, max_pin='x.x') }}
@@ -32,6 +31,7 @@ build:
 
 requirements:
   build:
+    - {{ stdlib('c') }}
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - patch       # [unix]
@@ -49,10 +49,10 @@ requirements:
     - perl
   host:
     - fontconfig {{ fontconfig }}  # [linux]
-    - krb5 1.20.1                  # [linux]
+    - krb5 {{ krb5 }}              # [linux]
     - libcups 2.4.2                # [linux]
     - dbus {{ dbus }}              # [unix]
-    - mysql 8.4.0                  # [unix]
+    - mysql {{ mysql }}            # [unix]
     - freetype {{ freetype }}      # [unix]
     - pcre2 {{ pcre2 }}            # [unix]
     - libglib {{ libglib }}        # [unix]
@@ -78,12 +78,9 @@ requirements:
     - xorg-libsm {{ xorg_libsm }}                    # [linux]
     - xorg-libx11 {{ xorg_libx11 }}                  # [linux]
   run:
-    - mesalib      # [linux]
-    - libcups 2.*  # [linux]
-    - mysql 8.4.*  # [unix]
-    - libpq
-    - openssl
-    - sqlite
+    - mesalib 25         # [linux]
+    - libcups 2.*        # [linux]
+    - mysql {{ mysql }}  # [unix]
   run_constrained:
     - qt-main >={{ version }},<7
     - qt >={{ version }},<7
@@ -146,6 +143,7 @@ outputs:
         - {{ pin_subpackage(name, max_pin='x.x') }}
     requirements:
       build:
+        - {{ stdlib('c') }}
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
       host:

